_rules/active-focus-shortcut-1e9941.md

```diff
@@ -0,0 +1,212 @@
+      }
+    </style>
+  </head>
+  <body onload="shortcut();">
+    <button id="italic" type="button" accessKey="i">Toggle italic format</button>
+    <div id="text">Some text inside the document content</div>
+  </body>
+</html>
+```
+
+#### Passed Example 3
+
+The [HTML document][] has a single [printable character][printable characters] [keyboard shortcut][keyboard shortcuts] for a [user interface component][], which is only available when that component has focus.
+
+```html
+<html>
+  <head>
+    <title>Passed Example 3</title>
+    <script>
+      function shortcut() {
+        document.body.addEventListener('keydown', function(event) {
+          const text = document.getElementById('text');
+
+          if (event.key === '+' && document.activeElement === text) {
+            document.getElementById('list').innerHTML += '<li>' + text + '</li>';
+            text.value = '';
+            event.preventDefault();
+          }
+        });
+      }
+    </script>
+  </head>
+  <body onload="shortcut();">
+    <label for="text">Add to list (press + to add):</label>
+    <input type="text" id="text">
+    <br>
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The [HTML document][] has a [keyboard shortcut][keyboard shortcuts] using only a [printable character][printable characters] for a [user interface component][], and it's available even when the component does not have focus.
+
+```html
+<html>
+  <head>
+    <title>Failed Example 1</title>
+    <script>
+      function shortcut() {
+        document.body.addEventListener('keydown', function(event) {
+          if (event.key === '+') {
+            event.preventDefault();
+
+            const text = document.getElementById('text');
+            document.getElementById('list').innerHTML += '<li>' + text.value + '</li>';
+            text.value = '';
+          }
+        });
+      }
+    </script>
+  </head>
+  <body onload="shortcut();">
+    <label for="text">Add to list (press + to add):</label>
+    <input type="text" id="text">
+    <br>
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The [HTML document][] does not use [keyboard shortcuts][keyboard shortcuts].
+
+```html
+<html>
+  <div>Document content</div>
+</html>
+```
+
+#### Inapplicable Example 2
+
+The document is not an [HTML document][].
+
+```html
+<svg height="100" width="100">
+  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
+  Sorry, your browser does not support inline SVG.  
+</svg>
+```
+
+[HTML document]: https://dom.spec.whatwg.org/#concept-document
+[keyboard shortcuts]: https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html#dfn-keyboard-shortcut
+[user interface component]: https://www.w3.org/WAI/WCAG21/Understanding/character-key-shortcuts.html#dfn-user-interface-component

```

Comment:
Link the defs in the global document, not in the specific Understanding one.
https://www.w3.org/TR/WCAG21/#dfn-keyboard-shortcuts
https://www.w3.org/TR/WCAG21/#dfn-user-interface-components